src/js/background.js

```diff
@@ -258,13 +258,17 @@ function dataLoaded(data) {
     }
 
     // Remove Firefox Preview
-    if (data.chooseProduct.indexOf('firefox-preview') >= 0) {
-        data.chooseProduct = data.chooseProduct.filter((v) => {
-            return v != 'firefox-preview';
-        });
-        browser.storage.local.set({
-            chooseProduct: data.chooseProduct
-        });
+    try {
+        if (data.chooseProduct.indexOf('firefox-preview') >= 0) {
+            data.chooseProduct = data.chooseProduct.filter((v) => {
+                return v != 'firefox-preview';
+            });
+            browser.storage.local.set({
+                chooseProduct: data.chooseProduct
+            });
+        }
+    } catch (error) {
+        console.log(error);
```

Comment:
We should leave the `catch` empty to avoid debugging logs in the production version.

Reply:
I'm torn on this. I'd much rather have it logged as a proper console.error(). I feel that using an empty catch to silently suppress an error is not good practice.

Reply:
The console.error instead of the console.log, was added to be more visible on the console, and also the part that verifies the 'firefox-preview' was removed.